build.sh

```diff
@@ -16,12 +16,12 @@ pushd iso
 
 	wget -nc "${base_url}/${latest_iso}" || die "Could not download iso"
 	wget -nc "${base_url}/${latest_iso}.DIGESTS.asc" || die "Could not download digests"
-	wget -nc "${base_url}/${latest_iso}.CONTENTS" || die "Could not download contents"
+	wget -nc "${base_url}/${latest_iso}.CONTENTS.gz" || die "Could not download contents"
 	sha512_digests=$(grep -A1 SHA512 "${iso}.DIGESTS.asc" | grep -v '^--')
 	gpg --verify "${iso}.DIGESTS.asc" || die "Insecure digests"
 	echo "${sha512_digests}" | sha512sum -c || die "Checksum validation failed"
 popd
-mkdir mnt
+mkdir -p mnt
```

Comment:
sorry, I'm not sure why you need this?

Reply:
consistency mostly I guess, since all other mkdirs use the -p option,
I also had some errors after that line and had to rerun a couple of times
(implementing a cleanup/env reset in the die function might be even better)

Reply:
if it's an issue, I can remove it from the PR

Reply:
I usually don't like to add unnecessary stuff so please remove it mate (sorry)

Reply:
no worries, updated the PR